src/library/autodj/dlgautodj.h

```diff
@@ -26,6 +26,7 @@ class DlgAutoDJ : public QWidget, public Ui::DlgAutoDJ, public LibraryView {
     virtual ~DlgAutoDJ();
 
     void onShow();
+    bool hasFocus() const;
```

Comment:
Ah yeah, I like to do that as well, I think that commit was one of the first times I touched C++.
I'll add a new commit once you're done reviewing